conda-store-server/conda_store_server/_internal/server/views/api.py

```diff
@@ -620,25 +631,77 @@ async def api_list_environments(
     status: Optional[schema.BuildStatus] = None,
     packages: Optional[List[str]] = Query([]),
     artifact: Optional[schema.BuildArtifactType] = None,
-    auth=Depends(dependencies.get_auth),
-    entity=Depends(dependencies.get_entity),
-    paginated_args=Depends(get_paginated_args),
-    conda_store=Depends(dependencies.get_conda_store),
+    jwt: Optional[str] = None,
+    auth: Authentication = Depends(dependencies.get_auth),
```

Comment:
Parameters with default values (even if optional) should come after those without.
Similiarly, optional should come after required ones.

Reply:
Whoops, I'll move these optional args below the `Depends` args.